danuluma/dannstore
# #161436296 add sales endpoints

#### What does this PR do?
Add endpoints to post and get sales record

#### Description of Task to be completed?
Add POST method to /sales
Add GET method to /sales

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Get access token  by sending a POST request to http://localhost:5000/dann/api/v2/login with the body of ```{"username":"owner", "password":"secret1"}````
* Use postman to send a POST request to http://localhost:5000/dann/api/v2/sales 
* The POST request should have just the book id in the body with the format of  ```{
        'book_id': int
    }```
* You may now send a GET request to http://localhost:5000/dann/api/v2/sales
* You should get a response containing the details of the sale record created

#### What are the relevant pivotal tracker stories?
[#161436296](https://www.pivotaltracker.com/story/show/161436296)
